docs/network protocol.md

```diff
@@ -176,6 +176,7 @@ Sent to clients purely to display a message to the player. While various message
 | receiving | int | ItemSend, ItemCheat, Hint | Destination player's ID |
 | item | [NetworkItem](#NetworkItem) | ItemSend, ItemCheat, Hint | Source player's ID, location ID, item ID and item flags |
 | found | bool | Hint | Whether the location hinted for was checked |
+| hint_status | [HintStatus](#HintStatus) | The status of the hint. |
```

Comment:
I think you're missing a column here (Message Types?).

![image](https://github.com/user-attachments/assets/5c683122-56a0-4081-8fd2-d1fbe33d9fd7)